sphinx_scality/static/css/current.css_t

```diff
@@ -384,27 +384,30 @@ div.body h6 {
 /* Admonition boxes */
 
 div.important {
-    background-color: {{ theme_importantboxbackground}};
+    background-color: {{ theme_boxbackground }};
```

Comment:
Isn't there a common class name / CSS selector to get _all_ box types? So you could share those `background-color` and `border-radius` properties (even `border` with a `border-color` override in subsequent rule definitions).

Reply:
```css
div.admonition {
  background-color: {{ theme_boxbackground }};
  border-radius: 4px;
  border: 1px solid red;
}

div.important {
  border-color: {{ theme_importantboxborder }};
}
```